.github/workflows/generate_package_wiki.yml

```
@@ -0,0 +1,37 @@
+name: Generate Package Wiki
+
+on:
+  schedule:
+    # Runs daily at a weird time (02:50 UTC) to avoid delays during periods of
+    # high loads of GitHub Actions workflow runs.
+    - cron: '50 2 * * *'  
+  workflow_dispatch:  # Allow manual triggering
+
+jobs:
+  generate-wiki:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

You should you should pin your GitHub Actions by commit-hash: https://blog.rafaelgss.dev/why-you-should-pin-actions-by-commit-hash (also few lines below)

```suggestion
        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```